content/deploy/cluster-sizing.md

```diff
@@ -1,29 +1,77 @@
+| PV IOPs               | 5000              |
+| PV Throughput (mbps)  | 500               |
+| Total Disk Space (TB) | 3                 |
+| Disk IOPS             | 5000              |
+| Disk MBps             | 500MBps           |
+
+### Rhize agent
+
+The Rhize agent typically runs on the edge, outside of the cluster entirely.
+For the Rhize Agent, the minimum recommended specifications are as follows:
+
+| Property              | Value |
+|-----------------------|-------|
+| CPU Speed (GHz)       | 2.8   |
+| vCPU per Node         | 2     |
+| Memory per node (GiB) | 1     |
+| Persisted volumes     | 12    |
+
+## Service-level recommendations
+
+The following table lists the **minimum** recommended specfications for the main services.
+Services with stateful PV have a persistent volume per pod.
+
+
+| Service                | Pods for HA (replica count) | vCPU per Pod | Memory Per Pod | Stateful PV | DiskSize (GiB) | Comments                                    |
+|------------------------|-----------------------------|--------------|----------------|-------------|----------------|---------------------------------------------|
+| `baas-alpha`           | 3                           | 8            | 16 (at least)  | Yes         | 750            | High throughput and IOPS                    |
+| `baas-zero`            | 3                           | 2            | 2              | Yes         | 350            | High throughput and IOPS                    |
+| `libre-core`           | 3                           | 1            | 2              | No          | N/A            |                                             |
+| `bpmn-engine`          | 3                           | 1            | 2              | No          | N/A            |                                             |
+| `nats`                 | 3                           | 1            | 2              | Yes         | 100GB          | High IOPS                                   |
+| `nats-box`             | 1                           | 0.25         |                | No          | N/A            |                                             |
+| `libre-audit`          | 2                           |              |                |             | N/A            |                                             |
+| `libre-audit-postgres` | 2                           |              |                | Yes         | 250            | runs in pod with `libre-audit`              |
+| `libre-ui`             | 3                           | 0.25         | 0.25           |             | N/A            |                                             |
+| `keycloak`             | 2                           | 1            | 2              |             | N/A            |                                             |
+| `keycloak-postgres`    | 2                           | 1            | 2              |             | 200            | runs in pod with `keycloak`                 |
```

Comment:
In the comment for this line, the first letter is not capitalized like the others.